zune-png/src/decoder.rs

```diff
@@ -777,7 +777,22 @@ impl<T: ZReaderTrait> PngDecoder<T> {
         {
             &mut out_u8
         } else {
-            let (a, b, c) = bytemuck::pod_align_to_mut::<u16, u8>(&mut out_u16);
+            #[inline(always)]
```

Comment:
This can be greatly simplified by just dropping all of this alignment logic. Converting a 16 slice to a u8 slice is always correct because the alignment of the target is smaller. Just casting it by hand and doubling the size is easier to understand than the align_to_mut dance imo.

Reply:
The bytemuck https://docs.rs/bytemuck/latest/bytemuck/fn.bytes_of_mut.html function could be used but if bytemuck is optional it probably makes more sense to just drop it